cmd/cofidectl/cmd/federation/federation.go

```diff
@@ -148,6 +155,19 @@ func checkFederationStatus(ctx context.Context, kubeConfig string, from *trust_z
 	return "Healthy", nil
 }
 
+// isTrustZoneDeployed returns whether a trust zone has been deployed, i.e. whether a SPIRE Helm release has been installed.
+func isTrustZoneDeployed(ctx context.Context, trustZone *trust_zone_proto.TrustZone) (bool, error) {
+	prov, err := helm.NewHelmSPIREProvider(ctx, trustZone, nil, nil)
+	if err != nil {
+		return false, err
+	}
+	if installed, err := prov.CheckIfAlreadyInstalled(); err != nil {
+		return false, err
+	} else {
+		return installed, nil
+	}
```

Comment:
I think we can just
```
return prov.CheckIfAlreadyInstalled()
``` 
directly here. Or am I missing an edge case?

Reply:
Right you are.